compose.ci.yml

```diff
@@ -6,10 +6,10 @@ services:
       - .env.ci
     environment:
       - TC_HOST=host.docker.internal
-      - POSTGRES_DB=gc_registry
+      - POSTGRES_DB=registry
       - POSTGRES_USER=postgres
       - POSTGRES_PASSWORD=password
-      - DATABASE_URL=postgresql://postgres:password@db_write/gc_registry
+      - DATABASE_URL=postgresql://postgres:password@db_write/write
```

Comment:
Not sure about this - have you checked that specifying a different str for `POSTGRES_DB` compared to the end of `DATABASE_URL` doesn't create two databases within the same container? Will test on my side too

Reply:
I don't think we are using `DATABASE_URL` as full connection string and now using host in it's place. This will go when merged.